antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/atn/ATNSerializer.kt

```diff
@@ -18,6 +17,9 @@ import org.antlr.v4.kotlinruntime.misc.IntervalSet
  */
 public open class ATNSerializer(public var atn: ATN) {
   public companion object {
+    public fun getSerialized(atn: ATN): IntegerList =
```

Comment:
Accidentally committed this too. It's just moving a public function over private ones, for consistency.